llvm/lib/CAS/MappedFileRegionBumpPtr.cpp

```diff
@@ -0,0 +1,340 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+/// \file Implements MappedFileRegionBumpPtr.
+///
+/// A bump pointer allocator, backed by a memory-mapped file.
```

Comment:
Shouldn't this be the Doxygen comment of `MappedFileRegionBumpPtr`?

Reply:
This is more like a comment for implementation details, not a user/API facing documentation. `class MappedFileRegionBumpPtr` always has a user facing Doxygen comment already.